medallion/exceptions.py

```diff
@@ -0,0 +1,46 @@
+# Medallion Custom Exceptions
+
+
+class MedallionError(Exception):
+    """base error class for Medallion
+
+    Args:
+        root_exception (Exception): Exception instance of root exception
+        desc (str): specific error description
+    """
+    def __init__(self, root_exception, desc=None):
+        self.root_exception = root_exception
+        self.desc = desc
+
+
+class ProcessingError(MedallionError):
+    """Internal processing error when processing user supplied data
+
+    Args:
+        root_exception (Exception): Exception instance of root exception
+        desc (str): specific error description
+    """
+    def __init__(self, root_exception, desc=None):
```

Comment:
We don't need to re-define `__init__` if we're just calling the superclass with the same arguments.

Reply:
Roger.